openpype/tools/publisher/control.py

```diff
@@ -176,11 +176,10 @@ def __init__(self, controller):
         self._create_discover_result = None
         self._convert_discover_result = None
         self._publish_discover_result = None
-        self._plugin_data = []
-        self._plugin_data_with_plugin = []
 
-        self._stored_plugins = set()
-        self._current_plugin_data = []
+        self._plugin_data_by_plugin_id = {}
+        self._current_plugin = None
+        self._current_plugin_data = {}
```

Comment:
This is still there solely to be 'faster' lookup?

Note that basically whenever `self._current_plugin is not None` then `self._current_plugin_data` should always be the same as: `self._plugin_data_by_plugin_id[self._current_plugin.id]`.